tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -38,6 +38,9 @@ ifdef(`KWD_PIPE_SCH_DEADLINE_US',`',`fatal_error(note: Need to define schedule f
 # define(DMIC_DAI_LINK_48k_NAME, `dmic01')
 ifdef(`DMIC_DAI_LINK_48k_NAME',`',define(DMIC_DAI_LINK_48k_NAME, `dmic01'))
 
+# define(DMIC_PIPELINE_48k_CORE_ID, 0)
+ifdef(`DMIC_PIPELINE_48k_CORE_ID',`',define(DMIC_PIPELINE_48k_CORE_ID, 0))
```

Comment:
Where is this being set in the Makefile ?
Btw, can you add `why` we need this change in the commit message. What feature are we enabling.

Reply:
Sure. Just added more description to the commit message.

DMIC_PIPELINE_48k_CORE_ID is set in `sof-tgl-max98373-rt5682.m4` and this line provides default value (=0) if the upper topology doesn't define it.